eslintrc.json

```diff
@@ -6,7 +6,8 @@
 
   "plugins": [
     "import",
-    "jest"
+    "jest",
+    "prettier"
```

Comment:
We need to explicitly list `prettier` as a plugin here because we reference a rule from it below